copulas/bivariate/independence.py

```diff
@@ -76,4 +76,4 @@ def percent_point(self, y, V):
 
         """
         self.check_fit()
-        return V
+        return y
```

Comment:
I'm pretty sure this is supposed to be `y`, not `V`. The entire point of the independence copula is that U and V are independent, so why is the inverse conditional CDF dependent on V?